junit-platform-console-standalone/junit-platform-console-standalone.gradle.kts

```diff
@@ -59,4 +74,10 @@ tasks {
 			))
 		}
 	}
+
+	// This jar contains some Java 9 code (I think from shadowed jansi)
+	// So in order to resolve this, it can only run on Java 9
+	verifyOSGiProperties {
+		property("-runee", "JavaSE-9")
+	}
```

Comment:
This feels wrong to me. Do you have a failure message?

Reply:
I will check to see exactly what classes are compiled with Java 9 bytecode (bnd uses the class' magic number to make this determination).

Reply:
The class in question is `org.junit.platform.console.ConsoleLauncherToolProvider` which implements `java.util.spi.ToolProvider` which is `@since 9` :)